packages/p2p/src/components/my-ads/ad-status.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react-lite';
+import { Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+import './ad-status.scss';
+
+const AdStatus = ({ days_until_archive, is_active }) => {
+    if (!is_active) {
+        return (
+            <div className='ad-status--inactive'>
+                <Text color='loss-danger' line_height='s' size='xs' weight='bold'>
+                    <Localize i18n_default_text='Inactive' />
+                </Text>
+            </div>
+        );
+    }
+
+    if (is_active && days_until_archive === 1) {
+        return (
+            <div className='ad-status--days_until_archive'>
```

Comment:
lines 21 to 28 seem to repeat in lines 34 to 41, Could we reuse this?

Reply:
The text in `Localize` is different, it cannot be reused.

Reply:
How about this?

```
const AdWarning = ({ days_until_archive }) =>  (
    <div className='ad-status--days_until_archive'>
        <Text color='warning' line_height='m' size='xs' weight='bold'>
            <Localize
                i18n_default_text={`Deactivating in: <br/> {{days_until_archive}} ${days_until_archive > 1 ? 'days' : 'day'}`}
                values={{ days_until_archive }}
            />
        </Text>
    </div>
);
```

Reply:
I don't see the point in complicating the text in the Localize. With translations, it's better to give the team the whole sentence and not separate parts of it because unlike English, sentences in other languages can be different with a change in one word also.